core/src/validator.rs

```diff
@@ -193,6 +194,7 @@ impl Default for ValidatorConfig {
             tpu_coalesce_ms: DEFAULT_TPU_COALESCE_MS,
             validator_exit: Arc::new(RwLock::new(ValidatorExit::default())),
             no_wait_for_vote_to_start_leader: true,
+            collect_per_program_timings: false,
```

Comment:
default it to `true`?

Reply:
I'll just remove it, if we need it for the cost model then we need to do it all the time anyway.